apps/web/src/components/web/tools/tool-actions.tsx

```
@@ -33,39 +64,157 @@ export const ToolActions = ({
-                <TriangleAlertIcon className="shrink-0 size-4 opacity-75" />
-                Report
-              </button>
-            </DropdownMenuItem>
-          </DropdownMenuContent>
-        </DropdownMenu>
-
-        <Tooltip tooltip={bookmarked ? "Remove bookmark" : "Add bookmark"}>
+
+            <DropdownMenuContent side="bottom" align="end" className="min-w-36">
+              <DropdownMenuItem asChild>
+                <button className={navLinkVariants()} onClick={() => setIsOpen(true)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)